grails-app/domain/org/pih/warehouse/order/OrderItemSummary.groovy

```diff
@@ -20,6 +20,12 @@ class OrderItemSummary implements Serializable {
     Integer quantityReceived
     Integer quantityCanceled
     Integer quantityInvoiced
+
+    Boolean itemOrdered = Boolean.FALSE
+    Boolean itemShipped = Boolean.FALSE
+    Boolean itemReceived = Boolean.FALSE
+    Boolean itemInvoiced = Boolean.FALSE
```

Comment:
it's rather a statement, not a change request, if you don't feel it's worth of doing - but when first reading this domain's code yesterday I found it difficult to understand what this `itemOrdered` mean being a boolean - I was "expecting" an object here, so maybe would it be worth to name it as `isItemOrdered` etc? I don't know if we have any convention regarding booleans?

Reply:
Yes, I was thinking about changing the name for this and for those from OrderSummary, because `itemOrdered` and `itemsOrdered` are too close. Will do it.